usaspending_api/api_contracts/contracts/v2/references/filter_tree/tas.md

```diff
@@ -0,0 +1,1035 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# TAS
+
+These endpoints are used to power USAspending.gov's TAS search component on the advanced search page.
+The response is a forest of filter search nodes, which despite having a unified structure represent different
+database fields based on depth in the tree.
+
+## Toptier Search [GET /api/v2/references/filter_tree/tas/{?depth}]
+
+Returns a list of toptier agencies that have at least one TAS affiliated with them
++ Request A request with a contract id
+    + Parameters
+        + `depth`: `0` (optional, enum[number]) How many levels deep the search will populate each tree. 
+            + Members
+                    + `0`
+                    + `1`
+                    + `2`
+        0 will return only agencies, 1 will return agencies and any federal accounts under them, and so on.
+    + Schema
+    
+            {
+            "$schema": "http://json-schema.org/draft-04/schema#",
+            "type": "string"
+            }
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + `results` (required, array[TASFilterTreeNode], fixed-type)
+    + Body
+
+            {
+            "results": [
+            {
+            "id": "012",
+            "ancestors": [],
+            "description": "Department of Agriculture",
+            "count": 139,
+            "children": null
+            }
+            ]
+            }
+
+
+## Search by Agency [GET /api/v2/references/filter_tree/tas/{agency}/{?depth}]
```

Comment:
Other contract files contain a single endpoint per file. Please split these into different files with appropriate directory structures and names to match the URL?

Reply:
As a heads up, our NAICS reference endpoint (https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_contracts/contracts/v2/references/naics.md) doesn't work that way.

If we plan to split both files, do you have a recommendation on how to name them? I see that we have an `awards/id.md` file, but that strikes me as very confusing to the unenlightened browser.

@MaxwellKendall @ebdabbs since these policies effect you most directly.

Reply:
I believe the existing pattern would make the files:

- /v2/references/filter_tree/tas.md
- v2/references/filter_tree/tas/agency.md
- v2/references/filter_tree/tas/agency/federal_account.md


We can't devote enough time to always accommodate the random "unenlightened browser."
There are currently two user-friendly methods to find the correct contract if the directory structure is confusing to someone:
1. Endpoints included in https://api.usaspending.gov/docs/endpoints contain links to the correct contract 
1. The search functionality in Github's U/I